### PR TITLE
Update Scalability-Tuning.md

### PR DESCRIPTION
**Pull request by bf8086**

### CODE REVIEW

**Comment by bf8086** [comment]
@calvinjia PTAL.

File: docs/en/advanced/Scalability-Tuning.md [hunk 1]
```diff
@@ -59,8 +59,8 @@ Increase the interval to reduce the number of heartbeat checks.
 
 The frequency in which a worker checks in with the master is set by the following properties:
 ```properties
-alluxio.worker.block.heartbeat.interval=60s
-alluxio.worker.filesystem.heartbeat.interval=60s
+alluxio.worker.block.heartbeat.interval=1s
+alluxio.worker.filesystem.heartbeat.interval=1s
```
Comment (calvinjia) [inline docs/en/advanced/Scalability-Tuning.md]: Should this be 1s or 1m?

Comment (bf8086) [reply docs/en/advanced/Scalability-Tuning.md]: The latest default value is `1sec`.

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**